CHANGELOG.md

```diff
@@ -2,6 +2,70 @@
 
 # Changelog
 
+## v1.26.0 (2022-05-04)
+
+    imagebuildah,build: move deepcopy of args before we spawn goroutine
+    Vendor in containers/storage v1.40.2
+    buildah.BuilderOptions.DefaultEnv is ignored, so mark it as deprecated
+    help output: get more consistent about option usage text
+    Handle OS version and features flags
+    buildah build: --annotation and --label should remove values
+    buildah build: add a --env
+    buildah: deep copy options.Args before performing concurrent build/stage
+    test: inline platform and builtinargs behaviour
+    vendor: bump imagebuilder to master/009dbc6
+    build: automatically set correct TARGETPLATFORM where expected
+    build(deps): bump github.com/fsouza/go-dockerclient
+    Vendor in containers/(common, storage, image)
+    imagebuildah, executor: process arg variables while populating baseMap
+    buildkit: add support for custom build output with --output
+    Cirrus: Update CI VMs to F36
+    fix staticcheck linter warning for deprecated function
+    Fix docs build on FreeBSD
+    build(deps): bump github.com/containernetworking/cni from 1.0.1 to 1.1.0
+    copier.unwrapError(): update for Go 1.16
+    copier.PutOptions: add StripSetuidBit/StripSetgidBit/StripStickyBit
+    copier.Put(): write to read-only directories
+    build(deps): bump github.com/cpuguy83/go-md2man/v2 in /tests/tools
+    Rename $TESTSDIR (the plural one), step 4 of 3
+    Rename $TESTSDIR (the plural one), step 3 of 3
+    Rename $TESTSDIR (the plural one), step 2 of 3
+    Rename $TESTSDIR (the plural one), step 1 of 3
```

Comment:
Nit: Too many similar lines with `Rename $TESTSDIR (the plural one), step 2 of 3` and has no context added to this. I think we can remove these lines.